Makefile

```diff
@@ -85,6 +85,9 @@ eslint-branch: node_modules/eslint node_modules/eslint-plugin-react node_modules
 i18n-lint:
 	@echo "$(JS_FILES)" | sed 's/\([^ ]*\/test\/[^ ]* *\)//g' | xargs -n1 $(I18NLINT)
 
+mixedindentlint: node_modules/mixedindentlint
+	@echo "$(JS_FILES)\n$(SASS_FILES)" | xargs mixedindentlint
```

Comment:
We may include it only for sass files and rely on `no-mixed-spaces-and-tabs` in eslint, since it’s a bit smarter about JS.